services/web/server/src/simcore_service_webserver/studies_dispatcher/settings.py

```diff
@@ -21,7 +21,7 @@ class StudiesDispatcherSettings(BaseCustomSettings):
     )
 
     STUDIES_DEFAULT_SERVICE_THUMBNAIL: HttpUrl = Field(
-        default="https://via.placeholder.com/170x120.png",
+        default=parse_obj_as(HttpUrl, "https://via.placeholder.com/170x120.png"),
```

Comment:
interestingly you went a different path here? It is the very same right?
I actually prefer this way than using cast. At least if it's wrong it will raise.